public/app/processing-overlay.css

```
@@ -1,271 +1,505 @@
+
 .proc-title {
-  font-family: 'DM Sans', system-ui, sans-serif;
-  font-size: 1.05rem;
-  font-weight: 700;
-  color: #e8eaf0;
-  letter-spacing: 0.015em;
-  margin-bottom: 0.35rem;
-  position: relative;
-  z-index: 1;
+  color: var(--vip-text);
+  font: 700 24px/1.15 'Inter', 'Segoe UI', sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)